lib/cf/bridge/src/index.js

```diff
@@ -293,7 +293,9 @@ const stopReporting = () => {
 };
 
 const scheduleUsageReporting = () => {
-  oauth.start('bridge', 'secret');
+  var clientId = process.env.CLIENT_ID ? process.env.CLIENT_ID : "bridge";
```

Comment:
I personally would prefer to have no hardcoded user/pass. If you go this direction you will need to modify the cf-bridge integration test in tests/cf/bridge